oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -52,6 +52,21 @@ This proposal focuses on long-term developer *Documentation* needs. It does not
 Decisions
 *********
 
+* We will use `reStructuredText (rST)`_ for all developer documentation (including in docstrings).  See the :ref:`Format Rationale` section below for more info.
```

Comment:
I thought there was a decision around Markdown for frontend repos somewhere. Not as a blocker to your update, but we should update this if that is the case. @davidjoy?

Reply:
Another place we've said will use Markdown are docstrings destined for Open API content.

Reply:
I have a more nuanced perspective on this after the hackathon... I attempted to see what it'd take to convert a bunch of our RST ReadTheDocs content to Markdown.  The answer is "a lot of custom code".

My take is:

- Developer Guides which have multiple pages, cross-reference other documentation, have complex tables, tables of contents, etc., should be in RST.  There's no substitute for it.  Even with modern Markdown-friendly tooling like Docusaurus, you have to do a bunch of extra work to get the same level of functionality.
- Repository READMEs can be in Markdown.  I'll hesitate to say "should", but I'm thinking it.  This sort of doc is less interconnected and doesn't constitute our formal platform documentation, doesn't get published to ReadTheDocs, etc. 
- Repositories that need to can use Markdown for API docs if their tooling supports it.  Paragon and frontend-platform, for instance, benefit greatly from the use of Markdown.  There's no RST equivalent for MDX (markdown files with embedded React components), and JSDoc speaks Markdown far more readily than RST.  
- IMHO, if ADRs and OEPs were in Markdown, we'd have more ADRs and OEPs.  This is possibly controversial, but I'll say it as a strawperson and see what we think.  These documents tend to stand alone, and tend to be simpler.  Markdown's lower barrier to entry and less opinionated syntax are probably a boon wherever we can get away with it.

Reply:
(The other part of the above breakdown is making sure we're actually putting documentation in the right places, which will inform what format is acceptable for it)

Reply:
1. READMEs are often published to readthedocs.
2. I don't really agree with the point around ADRs and OEPs. Not sure how much it is worth debating at this point.